components/HomeHeader.vue

```diff
@@ -0,0 +1,193 @@
+<template>
+  <swiper v-show="items.length > 0" class="swiper" :options="swiperOptions">
```

Comment:
Would a v-if be better here?

Reply:
Hm, yeah, no need to keep the swiper in DOM.